src/components/async-download-pdf/index.js

```diff
@@ -2,16 +2,17 @@
 import * as React from 'react'
 import {asyncComponent} from 'react-async-component'
 import DownloadPDF from './DownloadPDF'
-import {Spinning, Toast} from 'grommet'
-import {translate} from 'react-i18next'
+import {withTranslation} from 'react-i18next'
 
+// TODO Toast for error
 const ErrorNotification = ({t}) => (
-  <Toast status="critical">{t('unknownError')}</Toast>
+  <alert>{t('unknownError')}</alert>
```

Comment:
Does the new version of Grommet not have a Toast component?

Reply:
Good catch, I meant to go back and look for a replacement. It doesn't, maybe we could use https://github.com/schiehll/react-alert ?

Reply:
Sounds good

Reply:
I'm thinking that since this error completely blocks progress it would make more sense to have something that needs to be dismissed/acknowledged before moving on, like an alert or a modal, rather than a toaster. What do you think?

Reply:
That makes sense. I like a custom modal rather than a browser alert. I think it is a nicer experience

Reply:
Cool, I'll make this a modal component like the disclaimer and publish another commit.